dbcon/mysql/ha_mcs_execplan.cpp

```diff
@@ -6094,7 +6094,9 @@ bool isMCSTable(TABLE* table_ptr)
     string engineName = table_ptr->s->db_plugin->name.str;
 #endif
 
-    if (engineName == "Columnstore" || engineName == "InfiniDB")
+    if (engineName == "Columnstore" ||
+        engineName == "InfiniDB" ||
```

Comment:
There will be no Infinidb tables anymore.

Reply:
Good catch. This is updated.